src/status_im/contexts/wallet/send/transaction_confirmation/view.cljs

```diff
@@ -30,51 +30,54 @@
          (i18n/label :t/bridge)
          (i18n/label :t/send))]
       [quo/summary-tag
-       {:token        (if collectible? "" token-display-name)
-        :label        (str amount " " token-display-name)
-        :type         (if collectible? :collectible :token)
-        :image-source (if collectible? image-url :eth)}]]
+       (cond-> {:token (if collectible? "" token-display-name)
+                :label (str amount " " token-display-name)
+                :type  (if collectible? :collectible :token)}
+         collectible? (assoc :image-source image-url))]]
      (if (= transaction-type :tx/bridge)
-       (map-indexed
-        (fn [idx path]
-          (let [from-network             (:from path)
-                chain-id                 (:chain-id from-network)
-                network                  (rf/sub [:wallet/network-details-by-chain-id
-                                                  chain-id])
-                network-name             (:network-name network)
-                network-name-text        (name network-name)
-                network-name-capitalized (when (seq network-name-text)
-                                           (string/capitalize network-name-text))
-                network-color            (if (= network-name :mainnet) :ethereum network-name)]
-            [rn/view
-             {:style {:flex-direction :row
-                      :margin-top     4}}
-             (if (zero? idx)
-               [:<>
-                [quo/text
-                 {:size                :heading-1
-                  :weight              :semi-bold
-                  :style               style/title-container
-                  :accessibility-label :send-label}
-                 (i18n/label :t/from)]
-                [quo/summary-tag
-                 {:label               network-name-capitalized
-                  :type                :network
-                  :image-source        (:source network)
-                  :customization-color network-color}]]
-               [:<>
-                [quo/text
-                 {:size                :heading-1
-                  :weight              :semi-bold
-                  :style               style/title-container
-                  :accessibility-label :send-label}
-                 (str (i18n/label :t/and) " ")]
-                [quo/summary-tag
-                 {:label               network-name-capitalized
-                  :type                :network
-                  :image-source        (:source network)
-                  :customization-color network-color}]])]))
-        route)
+       (doall
+        (map-indexed
+         (fn [idx path]
+           (let [from-network             (:from path)
+                 chain-id                 (:chain-id from-network)
+                 network                  (rf/sub [:wallet/network-details-by-chain-id
+                                                   chain-id])
```

Comment:
the `rf/sub` has a `deref` hidden inside, and we were using it inside a lazy seq (returned by `map-indexed`) so we had a warning, adding a `doall` sovles the problem :)